dm_cli/import_package.py

```diff
@@ -41,17 +42,46 @@ def add_package_to_package(path: Path, package: Package) -> None:
     return add_package_to_package(Path(new_path), sub_folder)
 
 
-def import_package_tree(root_package: Package, data_source_id: str) -> None:
-    documents_to_upload: List[dict] = [root_package.to_dict()]
-    root_package.traverse_documents(lambda document, **kwargs: documents_to_upload.append(document))
-    root_package.traverse_package(lambda package: documents_to_upload.append(package.to_dict()))
+def import_root_package_tree(package: Package, data_source_id: str) -> None:
+    documents_to_upload: List[dict] = [package.to_dict()]
+    package.traverse_documents(lambda document, **kwargs: documents_to_upload.append(document))
+    package.traverse_package(lambda package: documents_to_upload.append(package.to_dict()))
 
     with IncrementalBar(
-        f"\tImporting {root_package.name}",
+        f"\tImporting {package.name}",
         max=len(documents_to_upload),
         suffix="%(percent).0f%% - [%(eta)ds/%(elapsed)ds]",
     ) as bar:
         for document in documents_to_upload:
             document = upload_blobs_in_document(document, data_source_id)
             dmss_api.document_add_simple(data_source_id, document)
         bar.next()
+
+
+def import_package_tree(package: Package, destination: str) -> None:
+    destination_parts = destination.split("/")
+    data_source = destination_parts[0]
+
+    documents_to_upload: List[dict] = []
+    if len(destination_parts) == 1:  # We're importing a root package
+        documents_to_upload.append(package.to_dict())
```

Comment:
would it make sense to call import_root_package_tree() here and end the import_package_tree() function? 

```suggestion
        import_root_package_tree(package, destiantion)
        return
```

Reply:
Oops! This if/else was added to be able to remove the "import_root_package_tree" function. Will fix